src/Evolution/Systems/Cce/Tags.hpp

```diff
@@ -0,0 +1,244 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/SpinWeighted.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+namespace Cce {
+
+/// Tags for Cauchy Characteristic Extraction routines
+namespace Tags {
+
+// Bondi parameter tags
+
+/// Bondi parameter \f$\beta\f$
+struct Beta : db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, 0>>;
+  static std::string name() noexcept { return "Beta"; }
+};
+
+/// \brief Bondi parameter \f$H = \partial_u J\f$.
+/// \note the notation in the literature is not consistent regarding this
+/// quantity, or whether it is denoted by an \f$H\f$ at all. The SpECTRE CCE
+/// module consistently uses it to describe the (retarded) partial time
+/// derivative of \f$J\f$ at fixed numerical radius \f$y\f$.
```

Comment:
as opposed to Bondi radius which has physical meaning but is not used as a numerical coordinate system. Is something to that effect worth including in the comment?

Reply:
I think so, yes. Or maybe `compactified radius`, if that's still a true statement? I was just confused because to me any radius is "numerical" if I put it on a grid :)